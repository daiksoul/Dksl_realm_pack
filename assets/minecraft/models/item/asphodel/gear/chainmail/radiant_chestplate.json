{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/chainmail_chestplate",
		"layer1": "misc/radiant_symbol"
	}
}
