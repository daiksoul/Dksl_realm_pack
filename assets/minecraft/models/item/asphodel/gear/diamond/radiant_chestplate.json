{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/diamond_chestplate",
		"layer1": "misc/radiant_symbol"
	}
}
