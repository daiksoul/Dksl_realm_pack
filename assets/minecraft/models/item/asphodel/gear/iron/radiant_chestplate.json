{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/iron_chestplate",
		"layer1": "misc/radiant_symbol"
	}
}
