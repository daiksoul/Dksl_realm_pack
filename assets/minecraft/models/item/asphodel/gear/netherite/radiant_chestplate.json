{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/netherite_chestplate",
		"layer1": "misc/radiant_symbol"
	}
}
