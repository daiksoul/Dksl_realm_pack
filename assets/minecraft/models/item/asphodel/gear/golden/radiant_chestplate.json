{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/golden_chestplate",
		"layer1": "misc/radiant_symbol"
	}
}
